### PR TITLE
terminate the process with exit code 0 if no violation is found

### PR DESCRIPTION
**Pull request by vbence86**
#### Expected
Process terminates with exit code 0 when no accessibility violations are detected.

#### Actual
It terminates with exit code 1 regardless.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "metal-a11y-checker",
-  "version": "1.0.2",
+  "version": "1.0.4",
```
Comment (jbalsas) [inline package.json]: This is going to conflict with the existing `1.0.3` version. Please, rebase first.

File: src/axe-runner.js [hunk 2]
```diff
@@ -65,12 +65,11 @@ function processReports(reports) {
...
       )
     );
+    process.exit(1);
   } else {
     log(chalk.green('No accessibility violations found in any packages'));
+    process.exit(0);
   }
-
-  if (violationsDetected) process.exit(1);
-  process.exit(0);
```
Comment (jbalsas) [inline src/axe-runner.js:73]: Why wasn't this working? :D

Comment (jbalsas) [reply src/axe-runner.js:73]: Didn't we simply want to change this for `sumViolations`?

Comment (vbence86) [reply src/axe-runner.js:73]: `violationsDetected` was an array and supposed to be checked against its length. I overlooked the mishap and never tested this eventuality again.

Comment (jbalsas) [reply src/axe-runner.js:73]: Not sure what that would mean as exit code means, but maybe even `process.exit(sumViolations)`?

Comment (vbence86) [reply src/axe-runner.js:73]: I'd keep terminating to process with exitCode = 1, I've got no prediction how Travis will handle exitCode > 1.

**Pull request closed without merging**